events/types.go

```diff
@@ -214,10 +214,11 @@ const (
 	Input
 
 	// Show is sent to widgets when their Scene is first shown to the user
-	// in its final form. Listening to this event enables widgets to perform
-	// initial one-time activities on startup, in the context of a fully
-	// rendered display. This is guaranteed to only happen once per widget
-	// per Scene.
+	// in its final form, and whenever a major content managing widget
+	// (e.g., [core.Tabs], [core.Pages]) shows a new element.
```

Comment:
We should probably mention the actual `core.WidgetBase.Shown` function that does that.